Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -112,7 +112,7 @@ public struct Configuration: Equatable {
     public init(path: String = Configuration.fileName, rootPath: String? = nil,
                 optional: Bool = true, quiet: Bool = false, enableAllRules: Bool = false, cachePath: String? = nil) {
         let fullPath: String
-        if let rootPath = rootPath {
+        if let rootPath = rootPath, rootPath.isDirectory() {
```

Comment:
I thought about using `rootPath.isSwiftFile()`, but I thought it'd be better to be safe.